### PR TITLE
Release/1.0.2

### PR DESCRIPTION
**Pull request by JasonMarechal25**

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
 Antares-Xpansion changelog
 =================
 
+v1.0.2 (01/2023)
```
Comment (tbittar) [inline docs/CHANGELOG.md:4]: Can we add bullet points before items to uniformize with the rest of the docs and mention that now we zip the entire output and not just the MPS ?

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+--------------------
+
+### Bug
+
+Fix missing performance graph from release
+
```
Comment (a-zakir) [inline docs/CHANGELOG.md:10]:
```suggestion
### Features
In full run mode (i.e with option --step full) xpansion output are zipped in antares archive
```

File: docs/CHANGELOG.md (hunk 1 above)
Comment (tbittar) [reply docs/CHANGELOG.md:4]: We will do this in the develop branch finally

**Review by tbittar — APPROVED** [review]

**Review by a-zakir — APPROVED** [review]

File: docs/CHANGELOG.md (hunk 2 above)
Comment (tbittar) [reply docs/CHANGELOG.md:10]: We will do this in develop

**Pull request merged**